htdocs/templates2/ocstyle/articles/EN/team.tpl

```diff
@@ -67,6 +67,12 @@
                 </td>
             </tr>
             <tr>
+                <td><img src="{$opt.page.teampic_url}dummy.jpg"/></td>
+                <td><a href="{$opt.page.teammember_url}viewprofile.php?userid=150360">following</a><br/>
+                    data maintenance, software development
```

Comment:
please only development like the other entries.

Reply:
Would be nice to check Slini as well ;)
It's same for him. @teiling88

Reply:
oh my fault.

Reply:
merged and deployed to live